etc/notebooks/sentiment_boilerplate_demo/sentiment_boilerplate_demo.ipynb

```diff
@@ -778,13 +778,15 @@
    "source": [
     "%%html\n",
     "<script>\n",
-    "    var channel = document.getElementById('defaultChannel');\n",
-    "    // Set the start and end for the timeline widget\n",
-    "    // Initialize the window values to some date string\n",
-    "    channel.set('window_end', new Date().toISOString());\n",
-    "    channel.set('window_start', new Date().toISOString());\n",
-    "    // The color palette used in the visualizations\n",
-    "    channel.set('palette', ['#2166ac','#b2182b', '#4393c3', '#d6604d', '#92c5de', '#f4a582', '#d1e5f0', '#fddbc7'])\n",
+    "    Urth.whenReady(function() {\n",
+    "        var channel = document.getElementById('defaultChannel');\n",
+    "        // Set the start and end for the timeline widget\n",
+    "        // Initialize the window values to some date string\n",
+    "        channel.set('window_end', new Date().toISOString());\n",
+    "        channel.set('window_start', new Date().toISOString());\n",
+    "        // The color palette used in the visualizations\n",
+    "        channel.set('palette', ['#2166ac','#b2182b', '#4393c3', '#d6604d', '#92c5de', '#f4a582', '#d1e5f0', '#fddbc7']) \n",
+    "    });\n",
```

Comment:
Won't all these changes also have to go into community sentiment which is a copy/paste duplicate of the more general template?